modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsPerformanceByStep.testcase

```diff
@@ -0,0 +1,274 @@
+
+	}
+
+	@description = "LPS-100674 - Verify that the default date format for en-US in Custom Range filter is MM/DD/YYYY"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test DefaultDateFormatIsMMDDYYYY {
+
+		// TODO LRQA-67691 DefaultDateFormatIsMMDDYYYY pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that the no completed item state is displaying in Performance by Step card (empty card)"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test EmptyStateIsDisplayedWhenThereIsNoCompletedItem {
+
+		// TODO LRQA-67688 EmptyStateIsDisplayedWhenThereIsNoCompletedItem pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that when the user selects Last 7 Days time period, the result and filter options displayed in accordance with reference"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test FilterItemsFromLast7Days {
+
+		// TODO LRQA-67687 FilterItemsFromLast7Days pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that when the user selects (Last 90 Days) time period the result and filter options displayed in accordance with reference"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test FilterItemsFromLast90Days {
+
+		// TODO LRQA-67685 FilterItemsFromLast90Days pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that when the user selects (Last 180 Days) time period the result and filter options are displayed in accordance with reference"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test FilterItemsFromLast180Days {
+
+		// TODO LRQA-67684 FilterItemsFromLast180Days pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that when the user selects (Last Year) time period the result and filter options displayed in accordance with reference"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test FilterItemsFromLastYear {
+
+		// TODO LRQA-67681 FilterItemsFromLastYear pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that when the user selects Today time period the result and filter options are displayed in accordance with reference"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test FilterItemsFromToday {
+
+		// TODO LRQA-67689 FilterItemsFromToday pending implementation
+
+	}
+
+	@description = "LPS-100679 - Verify that users are able to filter the Performance by Step page by period: Today, Yesterday, Last 7 Days, Last 30 Days, Last 90 Days, Last 180 Days, Last Year, Custom Range"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test FilterPerformanceByStepPagePerTimeRange {
+
+		// TODO LRQA-67667 FilterPerformanceByStepPagePerTimeRange pending implementation
+
+	}
+
+	@description = "LPS-98233 - Verify that the fields From and To are required"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test FromAndToAreRequiredFields {
+
+		// TODO LRQA-67686 FromAndToAreRequiredFields pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that hovering the info icon next to (Performance by Step) card will display a tooltip"
```

Comment:
Please, remove the parenthesis from (Perfomance by Step).
When, in the test map, you have double quotation marks in the test case description, you can only remove them from the stub description.